src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -557,7 +557,12 @@ public virtual string NormalizeUserName(string userName)
             {
                 return false;
             }
-            return await VerifyPasswordAsync(passwordStore, user, password, cancellationToken);
+            var result = await VerifyPasswordAsync(passwordStore, user, password, cancellationToken);
```

Comment:
Is there a way to hook convert this to IdentityResult. That way I insert logging here which can trace that the password was rehashed. Else we can look at logging to log for results other than IdentityResult as well